lib/memo_wise.rb

```diff
@@ -559,16 +585,18 @@ def reset_memo_wise(method_name = nil, *args, **kwargs)
 
   # Validates that {.memo_wise} has already been called on `method_name`.
   def validate_memo_wised!(method_name)
+    klass = instance_of?(Class) ? singleton_class : self.class
```

Comment:
Is it worth extracting this line into a `klass` method if we use it below too?

Reply:
Planning to do extraction and refactoring when I repeat this pattern for `reset_memo_wise` if that’s ok?